README.md

```diff
@@ -6,6 +6,16 @@
 
 An extension for [Asciidoctor.js](https://github.com/asciidoctor/asciidoctor.js) to convert diagrams to images using [Kroki](https://kroki.io)!
 
+  * [Install](#install)
```

Comment:
That's a shame that there's no `:toc:` alternative in Markdown :disappointed:

Reply:
My words...